gatsby-node.js

```diff
@@ -78,8 +78,12 @@ exports.onCreateNode = async ({
 }) => {
   const { createNodeField, createNode } = actions
   if (node.internal.type === `apps`) {
+
+    if( !node.imageUrl.trim()) ) return Promise.resolve()
```

Comment:
It should return the node with the extra field `lastCommit`

Reply:
but the url field is empty can we return without that? I have put a placeholder now and added the lastCommit field too